chat-server/src/main/java/pl/vyraj/ServerChat.java

```diff
@@ -0,0 +1,47 @@
+package pl.vyraj;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+public class ServerChat {
+    private final ServerSocket server;
+    private static final int PORT = 7777;
```

Comment:
Tego typu zmienne dobrze wczytywać jako zmienne środowiskowe lub tzw. propertiesy, natomiast oczywiście defaulty mogą pozostać

Reply:
Thanks for this remark. Code changed to read from application.properties file.